src/converter/frame.py

```diff
@@ -1,25 +1,26 @@
 import math
 from . import base, group, prototype, rectangle
 from converter import utils
-from sketchformat.layer_group import Artboard, SimpleGrid, LayoutGrid, Rect
+from sketchformat.layer_group import Frame, SimpleGrid, LayoutGrid, Rect
 from sketchformat.style import Fill, FillType
 from typing import Optional
 from collections import namedtuple
 
 
-def convert(fig_frame: dict) -> Artboard:
-    obj = Artboard(
+def convert(fig_frame: dict) -> Frame:
+    obj = Frame(
         **base.base_styled(fig_frame),
         **prototype.prototyping_information(fig_frame),
         grid=convert_grid(fig_frame),
+        groupBehavior=1,
```

Comment:
This is the magic ✨ In Sketch a frame is really a group with groupBehaviour set to 1